.github/dependabot.yml

```diff
@@ -11,8 +11,4 @@ updates:
       interval: "weekly"
     reviewers:
       - "aviv1620"
-    groups:
-      production-dependencies:
-        dependency-type: "production"
-      development-dependencies:
-        dependency-type: "development"
+    open-pull-requests-limit: 100
```

Comment:
That's a great idea, but don't think we have the capacity to manage 100 open pull request. Don't you think that 5 will be enough?
You can run npm audit and such locally to find things that should be upgraded

Reply:
The tests take very long time to complete.
And we have 42 dependencies updates.
(23 development-dependencies + 18 production-dependencies)

So I prefer at least capacity of 50 PRs to check all the updates with the same day.

![Screenshot 2024-03-20 174616](https://github.com/hasadna/open-bus-map-search/assets/44925669/19371fb1-16af-4a07-a16e-735a660449db)

Reply:
tests are running in parallel,
I don't see how anyone will go through 100 pull requests

Reply:
OK.
So how many PRs we want?

Reply:
I think 5 if optimal, but feel free to set it to any number that you think you'll be able to maintain
If we'll see we get too many pull requests that no-one is taking care of, I'll revert it back to 1.